templates/repo/header.tmpl

```diff
@@ -151,7 +151,7 @@
 {{end}}
 	<div class="ui tabs container">
 		{{if not (or .Repository.IsBeingCreated .Repository.IsBroken)}}
-			<div class="ui tabular stackable menu navbar">
+			<div class="ui tabular menu navbar gt-overflow-x-auto gt-overflow-y-hidden">
```

Comment:
<img width="1416" alt="Screen Shot 2023-04-28 at 15 23 28" src="https://user-images.githubusercontent.com/17645053/235081988-8dac3864-5eaf-4622-a35a-66ed99f6f27a.png">

The active header has a "bottom border" now, which is actually the top border of `ui tabs divider` , not very sure why, maybe because `overflow: auto/hidden` turned the menu div a [BFC](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context), and so the layout has changed?

Reply:
Forgot that. https://github.com/go-gitea/gitea/pull/24393/commits/3893b61f3c79d9f33a1b53756ed404377c4896c4


![image](https://user-images.githubusercontent.com/2114189/235088187-6ca0e100-855b-4eba-8ff3-038b7334886a.png)